pandas/core/generic.py

```diff
@@ -4275,7 +4275,29 @@ def get_ftype_counts(self):
 
     @property
     def dtypes(self):
-        """Return the dtypes in this object."""
+        """
+        Return the dtypes in this object.
+
+        Notes
+        -----
+        It returns a Series with the data type of each column.
+        If object contains data multiple dtypes in a single column,
+        dtypes will be chosen to accommodate all of the data types.
+        ``object`` is the most general.
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({'f': pd.np.random.rand(3),
```

Comment:
Can you just write out three floating point values? I'd like to avoid random data.

And FYI in general you don't want to use `pd.np`. You'll want to import NumPy manually (it's assumed to be imported in our docstrings.)

Reply:
Ah, and could you just make all these length-1 lists, just to be clearer? so `{'f': [1.0], 'i': [1], ...}`